metricbeat/main.go

```diff
@@ -27,7 +27,7 @@ package main
 import (
 	"os"
 
-	"github.com/elastic/beats/metricbeat/cmd"
+	"github.com/elastic/beats/x-pack/metricbeat/cmd"
```

Comment:
@sayden I don't think that should be here?

Reply:
Actually, it should, at least in this branch it was the way to go. I was struggling to make it work but maybe I can make a refactor to remove this (it is a bit obscure what it does even following the rabbit hole)

Reply:
we can't make the OSS metricbeat dependent on the x-pack metricbeat.